.editorconfig

```diff
@@ -0,0 +1,40 @@
+## EditorConfig is awesome: http://EditorConfig.org
+root = true
+
+
+## Newlines section
+
+# Make sure all files use lf as line ending
+[*]
+end_of_line = lf
+
+# Make sure our have an empty line at the end
+[*.{coffee,htaccess,html.twig,js,md,php}]
+insert_final_newline = true
+
+## Charset section
+[*.{coffee,htaccess,js,md,php,twig,xml,yml}]
+charset = utf-8
```

Comment:
Why not merge the empty line and charset blocks? The only difference I see is:

* newline => no xml and yml
* charset utf8 => no html

I think html could have charset utf8 and xml and yml files can contain final newlines without any problems.